README.md

```diff
@@ -107,6 +107,10 @@ The following payload will succeed.
 client-payload: '{"github": ${{ toJson(github) }}}'
 ```
 
+Additionally there is a limitation regarding the maximum length of strings, you are allowed to pass within `client-payload`.
+This might get into your way when you e.g. try to pass a Pull Request body.
+In this cases you might consider truncating the overlong strings, because otherwise your action will be interrupted with a `client_payload is too large` error.
+
```

Comment:
Thanks for contributing this!

It's not quite accurate that this error occurs on oversized property strings. The error occurs when the _total payload size_ hits a limit (although it was a bit difficult to figure out what the limit is, so I don't know the exact value). There might also be a string length limit on individual property values, but I've not seen it so far.

```suggestion
Additionally, there is a limitation on the total data size of the `client-payload`. A very large payload may result in a `client_payload is too large` error.
```